src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -34,15 +60,41 @@ private bool TryExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionC
                 return false;
             }
 
-            var formattingService = document.GetLanguageService<IEditorFormattingService>();
-            if (formattingService == null || !formattingService.SupportsFormatDocument)
+            if (!document.Project.Solution.Workspace.Options.GetOption(FeatureOnOffOptions.IsCodeCleanupRulesConfigured, LanguageNames.CSharp))
             {
-                return false;
+                ShowGoldBarForCodeCleanupConfiguration(document.Project.Solution.Workspace);
+                Format(args.TextView, document, null, context.WaitContext.UserCancellationToken);
```

Comment:
use named-param for 'null' to make code self-documenting.

Reply:
fixed